src/schema/validator.js

```diff
@@ -37,4 +37,14 @@ validator.addFormat('secureUrl', isSecureUrl);
 
 validator.addFormat('imageDataOrStrictRelativeUrl', imageDataOrStrictRelativeUrl);
 
-export default validator.compile(schemaObject);
+export const validateAddon = validator.compile({
+  ...schemaObject,
+  id: 'manifest',
+  $ref: '#/types/WebExtensionManifest',
+});
+
+export const validateLangPack = validator.compile({
+  ...schemaObject,
```

Comment:
Rather than use reuse the webextension manifest I think you could avoid updating the import code by creating a new schema for langpacks and just using that one.

```js
import langpackSchema from 'schema/langpack-manifest.json';
const langpackValidator = ajv({
  allErrors: true,
  errorDataPath: 'property',
  jsonPointers: true,
  verbose: true,
});
export const validateLangPack = validator.compile(langpackSchema);

Reply:
Good idea, but not necessary right now afaik. I might rethink that though when adding support for static themes or webext dictionaries.